Rebranch: Assertion failed for Windows x86 target SDK

### Description

When building the Swift toolchain for Windows from the rebranch, we hit an assertion failure when building the x86 runtime. From the assertion failure, and given the fact that x64 and arm64 runtime builds seem to build fine, I am guessing that the issue is 32-bits specific:

```
llvm::isUIntN(BitWidth, val) && "Value is not an N-bit unsigned value"
```

### Reproduction

Build the toolchain from the rebranch source for Windows

### Stack dump

```text
[281/388] Compiling S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation//WINDOWS/i686/Observation.obj
FAILED: [code=4294967295] stdlib/public/Observation/Sources/Observation/WINDOWS/i686/Observation.obj S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation/WINDOWS/i686/Observation.obj
stdlib\public\Observation\Sources\Observation\CMakeFiles\Observation.obj-c76dfb5.bat 7402b2ba62aa76c7
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\b\5\bin\swift-driver-new.exe'
S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/Observations.swift:208:28: warning: converting @isolated(any) function of type '@isolated(any) @Sendable () throws(Failure) -> Element' to synchronous function type '() throws(Failure) -> Element' is not allowed; this will be an error in a future Swift language mode

206 |         switch emit {

207 |         case .element(let element):

208 |           Result(catching: element).map { Iteration.next($0) }

    |                            `- warning: converting @isolated(any) function of type '@isolated(any) @Sendable () throws(Failure) -> Element' to synchronous function type '() throws(Failure) -> Element' is not allowed; this will be an error in a future Swift language mode

209 |         case .iteration(let iteration):

210 |           Result(catching: iteration)



Assertion failed: llvm::isUIntN(BitWidth, val) && "Value is not an N-bit unsigned value", file S:\SourceCache\llvm-project\llvm\include\llvm/ADT/APInt.h, line 128

Please submit a bug report (https://swift.org/contributing/#reporting-bugs) and include the crash backtrace.

Stack dump:

Command failed with exit code 4294967294
Command: 'S:\b\5\bin\swiftc.exe' -c -target i686-unknown-windows-msvc -resource-dir S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift -O -D SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY -D SWIFT_ENABLE_EXPERIMENTAL_DISTRIBUTED -D SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING -D SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING -D SWIFT_ENABLE_EXPERIMENTAL_OBSERVATION -D SWIFT_ENABLE_SYNCHRONIZATION -D SWIFT_ENABLE_VOLATILE -D SWIFT_RUNTIME_OS_VERSIONING -D SWIFT_STDLIB_ENABLE_UNICODE_DATA -D SWIFT_STDLIB_ENABLE_VECTOR_TYPES -D SWIFT_STDLIB_HAS_COMMANDLINE -D SWIFT_STDLIB_HAS_STDIN -D SWIFT_STDLIB_HAS_ENVIRON -Xcc -DSWIFT_STDLIB_HAS_ENVIRON -D SWIFT_CONCURRENCY_USES_DISPATCH -D SWIFT_STDLIB_OVERRIDABLE_RETAIN_RELEASE -D SWIFT_THREADING_WIN32 -module-cache-path S:/b/i686-unknown-windows-msvc/Runtime/./module-cache -no-link-objc-runtime -enable-library-evolution -library-level api -Xfrontend -require-explicit-availability=ignore -Xfrontend -enforce-exclusivity=unchecked -D SWIFT_ENABLE_REFLECTION -module-name Observation -swift-version 5 -Xfrontend -empty-abi-descriptor -runtime-compatibility-version none -disable-autolinking-runtime-compatibility-dynamic-replacements -Xfrontend -disable-autolinking-runtime-compatibility-concurrency -Xfrontend -disable-objc-interop -enable-experimental-feature NoncopyableGenerics2 -enable-experimental-feature SuppressedAssociatedTypes -enable-experimental-feature SE427NoInferenceOnExtension -enable-experimental-feature NonescapableTypes -enable-experimental-feature LifetimeDependence -enable-experimental-feature InoutLifetimeDependence -enable-experimental-feature LifetimeDependenceMutableAccessors -enable-upcoming-feature MemberImportVisibility -enable-experimental-feature Macros -enable-experimental-feature ExtensionMacros -Xfrontend -disable-implicit-string-processing-module-import -warn-implicit-overrides -Xfrontend -enable-lexical-lifetimes=false -Xfrontend -enable-ossa-modules -Xfrontend -disable-implicit-concurrency-module-import -Xfrontend -disable-implicit-string-processing-module-import -D_WINDLL -Xcc -Xclang -Xcc -ivfsoverlay -Xcc -Xclang -Xcc S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation/windows-sdk-vfs-overlay.yaml -vfsoverlay S:/b/i686-unknown-windows-msvc/Runtime/stdlib/windows-vfs-overlay.yaml -Xfrontend -strict-implicit-module-context -Xcc -Xclang -Xcc -fbuiltin-headers-in-system-modules -libc MultiThreadedDLL -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'SwiftCompatibilitySpan 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2, visionOS 1.0' -Xfrontend -define-availability -Xfrontend 'SwiftCompatibilitySpan 6.2:macOS 26.0, iOS 26.0, watchOS 26.0, tvOS 26.0, visionOS 26.0' -Xfrontend -target-min-inlining-version -Xfrontend min -module-link-name swiftObservation -whole-module-optimization -color-diagnostics -parse-as-library -I S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift/windows -o S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation//WINDOWS/i686/Observation.obj @S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation/e3167e865ebc7cee7ad5651331ee7fb178169948.txt
0.      Program arguments: C:\\Users\\steel\\src\\toolchain\\b\\5\\bin\\swiftc.exe -frontend -c S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/Locking.swift S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/Observable.swift S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/ObservationRegistrar.swift S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/ObservationTracking.swift S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/Observations.swift S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/ThreadLocal.swift -supplementary-output-file-map C:\\Users\\steel\\AppData\\Local\\Temp\\supplementaryOutputs-a2a4c0 -target i686-unknown-windows-msvc -disable-objc-interop -I S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift/windows -vfsoverlay S:/b/i686-unknown-windows-msvc/Runtime/stdlib/windows-vfs-overlay.yaml -color-diagnostics -enable-experimental-feature NoncopyableGenerics2 -enable-experimental-feature SuppressedAssociatedTypes -enable-experimental-feature SE427NoInferenceOnExtension -enable-experimental-feature NonescapableTypes -enable-experimental-feature LifetimeDependence -enable-experimental-feature InoutLifetimeDependence -enable-experimental-feature LifetimeDependenceMutableAccessors -enable-upcoming-feature MemberImportVisibility -enable-experimental-feature Macros -enable-experimental-feature ExtensionMacros -warn-implicit-overrides -enable-library-evolution -module-cache-path S:/b/i686-unknown-windows-msvc/Runtime/./module-cache -module-link-name swiftObservation -resource-dir S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift -swift-version 5 -O -library-level api -D SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY -D SWIFT_ENABLE_EXPERIMENTAL_DISTRIBUTED -D SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING -D SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING -D SWIFT_ENABLE_EXPERIMENTAL_OBSERVATION -D SWIFT_ENABLE_SYNCHRONIZATION -D SWIFT_ENABLE_VOLATILE -D SWIFT_RUNTIME_OS_VERSIONING -D SWIFT_STDLIB_ENABLE_UNICODE_DATA -D SWIFT_STDLIB_ENABLE_VECTOR_TYPES -D SWIFT_STDLIB_HAS_COMMANDLINE -D SWIFT_STDLIB_HAS_STDIN -D SWIFT_STDLIB_HAS_ENVIRON -D SWIFT_CONCURRENCY_USES_DISPATCH -D SWIFT_STDLIB_OVERRIDABLE_RETAIN_RELEASE -D SWIFT_THREADING_WIN32 -D SWIFT_ENABLE_REFLECTION -D _WINDLL -require-explicit-availability=ignore -enforce-exclusivity=unchecked -empty-abi-descriptor -disable-autolinking-runtime-compatibility-concurrency -disable-objc-interop -disable-implicit-string-processing-module-import -enable-lexical-lifetimes=false -enable-ossa-modules -disable-implicit-concurrency-module-import -disable-implicit-string-processing-module-import -strict-implicit-module-context -define-availability "SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999" -define-availability "StdlibDeploymentTarget 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999" -define-availability "SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2" -define-availability "StdlibDeploymentTarget 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2" -define-availability "SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0" -define-availability "StdlibDeploymentTarget 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0" -define-availability "SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4" -define-availability "StdlibDeploymentTarget 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4" -define-availability "SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0" -define-availability "StdlibDeploymentTarget 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0" -define-availability "SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5" -define-availability "StdlibDeploymentTarget 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5" -define-availability "SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0" -define-availability "StdlibDeploymentTarget 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0" -define-availability "SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4" -define-availability "StdlibDeploymentTarget 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4" -define-availability "SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0" -define-availability "StdlibDeploymentTarget 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0" -define-availability "SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4" -define-availability "StdlibDeploymentTarget 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4" -define-availability "SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0" -define-availability "StdlibDeploymentTarget 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0" -define-availability "SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1" -define-availability "StdlibDeploymentTarget 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1" -define-availability "SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0" -define-availability "StdlibDeploymentTarget 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0" -define-availability "SwiftStdlib 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4" -define-availability "StdlibDeploymentTarget 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4" -define-availability "SwiftStdlib 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "StdlibDeploymentTarget 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "SwiftStdlib 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "StdlibDeploymentTarget 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "SwiftCompatibilitySpan 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2, visionOS 1.0" -define-availability "SwiftCompatibilitySpan 6.2:macOS 26.0, iOS 26.0, watchOS 26.0, tvOS 26.0, visionOS 26.0" -target-min-inlining-version min -in-process-plugin-server-path C:\\Users\\steel\\src\\toolchain\\b\\5\\bin\\SwiftInProcPluginServer.dll -plugin-path C:\\Users\\steel\\src\\toolchain\\b\\5\\bin -Xcc -DSWIFT_STDLIB_HAS_ENVIRON -Xcc -Xclang -Xcc -ivfsoverlay -Xcc -Xclang -Xcc S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation/windows-sdk-vfs-overlay.yaml -Xcc -Xclang -Xcc -fbuiltin-headers-in-system-modules -autolink-library oldnames -autolink-library msvcrt -Xcc -D_MT -Xcc -D_DLL -parse-as-library -module-name Observation -o S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Observation/Sources/Observation//WINDOWS/i686/Observation.obj -runtime-compatibility-version none -disable-autolinking-runtime-compatibility-dynamic-replacements

1.      Swift version 6.2-dev (LLVM b0a5cdcad00b30f, Swift dd04814699722b8)

2.      Compiling with effective version 5.10

3.      While evaluating request IRGenRequest(IR Generation for module Observation)

4.      While emitting IR SIL function "@$s11Observation12ObservationsV8IteratorV4next9isolationxSgScA_pSgYi_tYaq_YKF".

 for 'next(isolation:)' (at S:/SourceCache/swift/stdlib/public/Observation/Sources/Observation/Observations.swift:245:21)

Exception Code: 0x80000003

 #0 0x00007ff63b0198b5 HandleAbort S:\SourceCache\llvm-project\llvm\lib\Support\Windows\Signals.inc:372:0

 #1 0x00007ffb08bf1989 (C:\WINDOWS\System32\ucrtbase.dll+0xc1989)

 #2 0x00007ffb08bd4ab1 (C:\WINDOWS\System32\ucrtbase.dll+0xa4ab1)

 #3 0x00007ffb08bf2986 (C:\WINDOWS\System32\ucrtbase.dll+0xc2986)

 #4 0x00007ffb08bf2b61 (C:\WINDOWS\System32\ucrtbase.dll+0xc2b61)

 #5 0x00007ff63485343b `anonymous namespace'::emitIndirectAsyncFunctionPointer S:\SourceCache\swift\lib\IRGen\GenCall.cpp:2559:0

 #6 0x00007ff6348435f8 `swift::irgen::getAsyncFunctionAndSize'::`2'::<lambda_1>::operator() S:\SourceCache\swift\lib\IRGen\GenCall.cpp:2638:0

 #7 0x00007ff634861915 swift::irgen::getAsyncFunctionAndSize(class swift::irgen::IRGenFunction &, class swift::irgen::FunctionPointer, struct std::pair<bool, bool>) S:\SourceCache\swift\lib\IRGen\GenCall.cpp:2683:0

 #8 0x00007ff634848e21 llvm::Twine::{ctor} S:\SourceCache\llvm-project\llvm\include\llvm\ADT\Twine.h:279:0

 #9 0x00007ff634848e21 `anonymous namespace'::AsyncCallEmission::begin S:\SourceCache\swift\lib\IRGen\GenCall.cpp:3154:0

#10 0x00007ff63497aad6 `anonymous namespace'::IRGenSILFunction::visitFullApplySite S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:3857:0

#11 0x00007ff63496e7ea swift::SILInstructionVisitor<`anonymous namespace'::IRGenSILFunction,void>::visit S:\SourceCache\swift\include\swift\SIL\SILNodes.def:590:0

#12 0x00007ff634983e6c `anonymous namespace'::IRGenSILFunction::visitSILBasicBlock S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:2845:0

#13 0x00007ff63495faef `anonymous namespace'::IRGenSILFunction::emitSILFunction S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:2709:0

#14 0x00007ff63495eaee swift::irgen::IRGenModule::emitSILFunction(class swift::SILFunction *) S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:2550:0

#15 0x00007ff6347a13de swift::irgen::CurrentIGMPtr::{dtor} S:\SourceCache\swift\lib\IRGen\IRGenModule.h:2061:0

#16 0x00007ff6347a13de swift::irgen::IRGenerator::emitGlobalTopLevel(class std::vector<class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>, class std::allocator<class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>>> const &) S:\SourceCache\swift\lib\IRGen\GenDecl.cpp:1235:0

#17 0x00007ff63472b5b1 std::vector<std::basic_string<char,std::char_traits<char>,std::allocator<char> >,std::allocator<std::basic_string<char,std::char_traits<char>,std::allocator<char> > > >::{dtor} C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.44.35207\include\vector:830:0

#18 0x00007ff63472b5b1 swift::IRGenRequest::evaluate(class swift::Evaluator &, struct swift::IRGenDescriptor) const S:\SourceCache\swift\lib\IRGen\IRGen.cpp:1406:0

#19 0x00007ff634734966 swift::SimpleRequest<class swift::IRGenRequest, (struct swift::IRGenDescriptor), 17>::evaluateRequest(class swift::IRGenRequest const &, class swift::Evaluator &) S:\SourceCache\swift\include\swift\AST\SimpleRequest.h:313:0

#20 0x00007ff6347253d7 swift::Evaluator::getResultUncached<class swift::IRGenRequest, class `class swift::GeneratedModule __cdecl swift::evaluateOrFatal<class swift::IRGenRequest>(class swift::Evaluator &, class swift::IRGenRequest)'::`2'::<lambda_1>>(class swift::IRGenRequest const &, class `class swift::GeneratedModule __cdecl swift::evaluateOrFatal<class swift::IRGenRequest>(class swift::Evaluator &, class swift::IRGenRequest)'::`2'::<lambda_1>) S:\SourceCache\swift\include\swift\AST\Evaluator.h:347:0

#21 0x00007ff63472eb91 swift::performIRGeneration(class swift::ModuleDecl *, class swift::IRGenOptions const &, struct swift::TBDGenOptions const &, class std::unique_ptr<class swift::SILModule, struct std::default_delete<class swift::SILModule>>, class llvm::StringRef, class swift::PrimarySpecificPaths const &, class llvm::ArrayRef<class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>>, class llvm::GlobalVariable **) S:\SourceCache\swift\lib\IRGen\IRGen.cpp:1824:0

#22 0x00007ff63438de42 generateIR S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:1785:0

#23 0x00007ff63438de42 performCompileStepsPostSILGen S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:2139:0

#24 0x00007ff63438e3ca std::unique_ptr<swift::SILModule,std::default_delete<swift::SILModule> >::{dtor} C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.44.35207\include\memory:3426:0

#25 0x00007ff63438e3ca swift::performCompileStepsPostSema(class swift::CompilerInstance &, int &, class swift::FrontendObserver *) S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:777:0

#26 0x00007ff63438d42a performAction S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:1348:0

#27 0x00007ff63438d6db performCompile S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:1734:0

#28 0x00007ff63438f845 swift::performFrontend(class llvm::ArrayRef<char const *>, char const *, void *, class swift::FrontendObserver *) S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:2454:0

#29 0x00007ff6341cda5d run_driver S:\SourceCache\swift\lib\DriverTool\driver.cpp:443:0

#30 0x00007ff6341cd4b7 swift::mainEntry(int, char const **) S:\SourceCache\swift\lib\DriverTool\driver.cpp:530:0

#31 0x00007ff63b082b50 invoke_main D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:78:0

#32 0x00007ff63b082b50 __scrt_common_main_seh D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288:0

#33 0x00007ffb0ac3e8d7 (C:\WINDOWS\System32\KERNEL32.DLL+0x2e8d7)

#34 0x00007ffb0b97c34c (C:\WINDOWS\SYSTEM32\ntdll.dll+0x3c34c)

<unknown>:0: error: compile command failed due to signal -2147483645 (use -v to see invocation)
Batch file failed at line 4 with errorcode -1
[282/388] Generating S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift/windows/Synchronization.swiftmodule/i686-unknown-windows-msvc.swiftmodule
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\b\5\bin\swift-driver-new.exe'
[283/388] Compiling S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Differentiation//WINDOWS/i686/_Differentiation.obj
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\b\5\bin\swift-driver-new.exe'
[284/388] Compiling S:/b/i686-unknown-windows-msvc/Runtime/stdlib/private/SwiftPrivate//WINDOWS/i686/SwiftPrivate.obj
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\b\5\bin\swift-driver-new.exe'
[285/388] Compiling S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Synchronization//WINDOWS/i686/Synchronization.obj
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\b\5\bin\swift-driver-new.exe'
[286/388] Compiling S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency//WINDOWS/i686/_Concurrency.obj
FAILED: [code=4294967295] stdlib/public/Concurrency/WINDOWS/i686/_Concurrency.obj S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/WINDOWS/i686/_Concurrency.obj
stdlib\public\Concurrency\CMakeFiles\_Concurrency.obj-7e1c7bc.bat 1a9b58634cace7d1
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\b\5\bin\swift-driver-new.exe'
Assertion failed: llvm::isUIntN(BitWidth, val) && "Value is not an N-bit unsigned value", file S:\SourceCache\llvm-project\llvm\include\llvm/ADT/APInt.h, line 128

Please submit a bug report (https://swift.org/contributing/#reporting-bugs) and include the crash backtrace.

Stack dump:

0.      Program arguments: C:\\Users\\steel\\src\\toolchain\\b\\5\\bin\\swiftc.exe -frontend -c S:/SourceCache/swift/stdlib/public/Concurrency/Actor.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncLet.swift S:/SourceCache/swift/stdlib/public/Concurrency/CheckedContinuation.swift S:/SourceCache/swift/stdlib/public/Concurrency/Errors.swift S:/SourceCache/swift/stdlib/public/Concurrency/Executor.swift S:/SourceCache/swift/stdlib/public/Concurrency/ExecutorBridge.swift S:/SourceCache/swift/stdlib/public/Concurrency/ExecutorAssertions.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncCompactMapSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncDropFirstSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncDropWhileSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncFilterSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncFlatMapSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncIteratorProtocol.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncMapSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncPrefixSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncPrefixWhileSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingCompactMapSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingDropWhileSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingFilterSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingFlatMapSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingMapSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingPrefixWhileSequence.swift S:/SourceCache/swift/stdlib/public/Concurrency/PartialAsyncTask.swift S:/SourceCache/swift/stdlib/public/Concurrency/GlobalActor.swift S:/SourceCache/swift/stdlib/public/Concurrency/GlobalConcurrentExecutor.swift S:/SourceCache/swift/stdlib/public/Concurrency/MainActor.swift S:/SourceCache/swift/stdlib/public/Concurrency/PriorityQueue.swift S:/SourceCache/swift/stdlib/public/Concurrency/SourceCompatibilityShims.swift S:/SourceCache/swift/stdlib/public/Concurrency/Task.swift S:/SourceCache/swift/stdlib/public/Concurrency/Task+PriorityEscalation.swift S:/SourceCache/swift/stdlib/public/Concurrency/Task+TaskExecutor.swift S:/SourceCache/swift/stdlib/public/Concurrency/TaskCancellation.swift S:/SourceCache/swift/stdlib/public/Concurrency/TaskGroup.swift S:/SourceCache/swift/stdlib/public/Concurrency/DiscardingTaskGroup.swift S:/SourceCache/swift/stdlib/public/Concurrency/TaskLocal.swift S:/SourceCache/swift/stdlib/public/Concurrency/TaskSleep.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncStreamBuffer.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncStream.swift S:/SourceCache/swift/stdlib/public/Concurrency/AsyncThrowingStream.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/_DequeBuffer.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/_DequeBufferHeader.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/_DequeSlot.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/_UnsafeWrappedBuffer.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Compatibility.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Storage.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+UnsafeHandle.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Codable.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Collection.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+CustomDebugStringConvertible.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+CustomReflectable.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+CustomStringConvertible.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Equatable.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+ExpressibleByArrayLiteral.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Extras.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Hashable.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/Deque+Testing.swift S:/SourceCache/swift/stdlib/public/Concurrency/Deque/UnsafeMutableBufferPointer+Utilities.swift S:/SourceCache/swift/stdlib/public/Concurrency/Clock.swift S:/SourceCache/swift/stdlib/public/Concurrency/ContinuousClock.swift S:/SourceCache/swift/stdlib/public/Concurrency/SuspendingClock.swift S:/SourceCache/swift/stdlib/public/Concurrency/TaskSleepDuration.swift S:/SourceCache/swift/stdlib/public/Concurrency/UnimplementedExecutor.swift S:/SourceCache/swift/stdlib/public/Concurrency/CooperativeExecutor.swift S:/SourceCache/swift/stdlib/public/Concurrency/PlatformExecutorDarwin.swift S:/SourceCache/swift/stdlib/public/Concurrency/PlatformExecutorLinux.swift S:/SourceCache/swift/stdlib/public/Concurrency/PlatformExecutorWindows.swift S:/SourceCache/swift/stdlib/public/Concurrency/PlatformExecutorOpenBSD.swift S:/SourceCache/swift/stdlib/public/Concurrency/PlatformExecutorFreeBSD.swift S:/SourceCache/swift/stdlib/public/Concurrency/DispatchExecutor.swift S:/SourceCache/swift/stdlib/public/Concurrency/CFExecutor.swift S:/SourceCache/swift/stdlib/public/Concurrency/ExecutorImpl.swift S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/4/Task+init.swift S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/4/TaskGroup+addTask.swift S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/4/Task+immediate.swift -supplementary-output-file-map C:\\Users\\steel\\AppData\\Local\\Temp\\supplementaryOutputs-87d596 -disable-objc-attr-requires-foundation-module -target i686-unknown-windows-msvc -disable-objc-interop -I S:/SourceCache/swift/stdlib/public/Concurrency/InternalShims -I S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift/windows -vfsoverlay S:/b/i686-unknown-windows-msvc/Runtime/stdlib/windows-vfs-overlay.yaml -color-diagnostics -enable-experimental-feature NoncopyableGenerics2 -enable-experimental-feature SuppressedAssociatedTypes -enable-experimental-feature SE427NoInferenceOnExtension -enable-experimental-feature NonescapableTypes -enable-experimental-feature LifetimeDependence -enable-experimental-feature InoutLifetimeDependence -enable-experimental-feature LifetimeDependenceMutableAccessors -enable-upcoming-feature MemberImportVisibility -enable-experimental-feature IsolatedAny -enable-experimental-feature AllowUnsafeAttribute -enable-experimental-feature Extern -strict-memory-safety -warn-implicit-overrides -enable-library-evolution -module-cache-path S:/b/i686-unknown-windows-msvc/Runtime/./module-cache -module-link-name swift_Concurrency -parse-stdlib -resource-dir S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift -swift-version 5 -O -library-level api -D SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY -D SWIFT_ENABLE_EXPERIMENTAL_DISTRIBUTED -D SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING -D SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING -D SWIFT_ENABLE_EXPERIMENTAL_OBSERVATION -D SWIFT_ENABLE_SYNCHRONIZATION -D SWIFT_ENABLE_VOLATILE -D SWIFT_RUNTIME_OS_VERSIONING -D SWIFT_STDLIB_ENABLE_UNICODE_DATA -D SWIFT_STDLIB_ENABLE_VECTOR_TYPES -D SWIFT_STDLIB_HAS_COMMANDLINE -D SWIFT_STDLIB_HAS_STDIN -D SWIFT_STDLIB_HAS_ENVIRON -D SWIFT_CONCURRENCY_USES_DISPATCH -D SWIFT_STDLIB_OVERRIDABLE_RETAIN_RELEASE -D SWIFT_THREADING_WIN32 -D SWIFT_ENABLE_REFLECTION -D _WINDLL -require-explicit-availability=ignore -enforce-exclusivity=unchecked -empty-abi-descriptor -disable-autolinking-runtime-compatibility-concurrency -disable-objc-interop -enable-lexical-lifetimes=false -enable-ossa-modules -disable-implicit-concurrency-module-import -disable-implicit-string-processing-module-import -strict-implicit-module-context -define-availability "SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999" -define-availability "StdlibDeploymentTarget 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999" -define-availability "SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2" -define-availability "StdlibDeploymentTarget 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2" -define-availability "SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0" -define-availability "StdlibDeploymentTarget 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0" -define-availability "SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4" -define-availability "StdlibDeploymentTarget 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4" -define-availability "SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0" -define-availability "StdlibDeploymentTarget 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0" -define-availability "SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5" -define-availability "StdlibDeploymentTarget 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5" -define-availability "SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0" -define-availability "StdlibDeploymentTarget 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0" -define-availability "SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4" -define-availability "StdlibDeploymentTarget 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4" -define-availability "SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0" -define-availability "StdlibDeploymentTarget 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0" -define-availability "SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4" -define-availability "StdlibDeploymentTarget 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4" -define-availability "SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0" -define-availability "StdlibDeploymentTarget 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0" -define-availability "SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1" -define-availability "StdlibDeploymentTarget 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1" -define-availability "SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0" -define-availability "StdlibDeploymentTarget 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0" -define-availability "SwiftStdlib 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4" -define-availability "StdlibDeploymentTarget 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4" -define-availability "SwiftStdlib 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "StdlibDeploymentTarget 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "SwiftStdlib 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "StdlibDeploymentTarget 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999" -define-availability "SwiftCompatibilitySpan 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2, visionOS 1.0" -define-availability "SwiftCompatibilitySpan 6.2:macOS 26.0, iOS 26.0, watchOS 26.0, tvOS 26.0, visionOS 26.0" -target-min-inlining-version min -in-process-plugin-server-path C:\\Users\\steel\\src\\toolchain\\b\\5\\bin\\SwiftInProcPluginServer.dll -plugin-path C:\\Users\\steel\\src\\toolchain\\b\\5\\bin -Xcc -DSWIFT_STDLIB_HAS_ENVIRON -Xcc -Xclang -Xcc -ivfsoverlay -Xcc -Xclang -Xcc S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/windows-sdk-vfs-overlay.yaml -Xcc -Xclang -Xcc -fbuiltin-headers-in-system-modules -autolink-library oldnames -autolink-library msvcrt -Xcc -D_MT -Xcc -D_DLL -parse-as-library -module-name _Concurrency -o S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency//WINDOWS/i686/_Concurrency.obj -runtime-compatibility-version none -disable-autolinking-runtime-compatibility-dynamic-replacements
Command failed with exit code 4294967294
Command: 'S:\b\5\bin\swiftc.exe' -c -target i686-unknown-windows-msvc -resource-dir S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift -O -D SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY -D SWIFT_ENABLE_EXPERIMENTAL_DISTRIBUTED -D SWIFT_ENABLE_EXPERIMENTAL_DIFFERENTIABLE_PROGRAMMING -D SWIFT_ENABLE_EXPERIMENTAL_STRING_PROCESSING -D SWIFT_ENABLE_EXPERIMENTAL_OBSERVATION -D SWIFT_ENABLE_SYNCHRONIZATION -D SWIFT_ENABLE_VOLATILE -D SWIFT_RUNTIME_OS_VERSIONING -D SWIFT_STDLIB_ENABLE_UNICODE_DATA -D SWIFT_STDLIB_ENABLE_VECTOR_TYPES -D SWIFT_STDLIB_HAS_COMMANDLINE -D SWIFT_STDLIB_HAS_STDIN -D SWIFT_STDLIB_HAS_ENVIRON -Xcc -DSWIFT_STDLIB_HAS_ENVIRON -D SWIFT_CONCURRENCY_USES_DISPATCH -D SWIFT_STDLIB_OVERRIDABLE_RETAIN_RELEASE -D SWIFT_THREADING_WIN32 -module-cache-path S:/b/i686-unknown-windows-msvc/Runtime/./module-cache -no-link-objc-runtime -enable-library-evolution -library-level api -Xfrontend -require-explicit-availability=ignore -Xfrontend -enforce-exclusivity=unchecked -D SWIFT_ENABLE_REFLECTION -module-name _Concurrency -swift-version 5 -Xfrontend -empty-abi-descriptor -runtime-compatibility-version none -disable-autolinking-runtime-compatibility-dynamic-replacements -Xfrontend -disable-autolinking-runtime-compatibility-concurrency -Xfrontend -disable-objc-interop -enable-experimental-feature NoncopyableGenerics2 -enable-experimental-feature SuppressedAssociatedTypes -enable-experimental-feature SE427NoInferenceOnExtension -enable-experimental-feature NonescapableTypes -enable-experimental-feature LifetimeDependence -enable-experimental-feature InoutLifetimeDependence -enable-experimental-feature LifetimeDependenceMutableAccessors -enable-upcoming-feature MemberImportVisibility -parse-stdlib -IS:/SourceCache/swift/stdlib/public/Concurrency/InternalShims -enable-experimental-feature IsolatedAny -strict-memory-safety -enable-experimental-feature AllowUnsafeAttribute -enable-experimental-feature Extern -warn-implicit-overrides -Xfrontend -enable-lexical-lifetimes=false -Xfrontend -enable-ossa-modules -Xfrontend -disable-implicit-concurrency-module-import -Xfrontend -disable-implicit-string-processing-module-import -D_WINDLL -Xcc -Xclang -Xcc -ivfsoverlay -Xcc -Xclang -Xcc S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/windows-sdk-vfs-overlay.yaml -vfsoverlay S:/b/i686-unknown-windows-msvc/Runtime/stdlib/windows-vfs-overlay.yaml -Xfrontend -strict-implicit-module-context -Xcc -Xclang -Xcc -fbuiltin-headers-in-system-modules -libc MultiThreadedDLL -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 9999:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.1:macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.2:macOS 10.15.4, iOS 13.4, watchOS 6.2, tvOS 13.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.3:macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.4:macOS 11.3, iOS 14.5, watchOS 7.4, tvOS 14.5' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.5:macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.6:macOS 12.3, iOS 15.4, watchOS 8.5, tvOS 15.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.7:macOS 13.0, iOS 16.0, watchOS 9.0, tvOS 16.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.8:macOS 13.3, iOS 16.4, watchOS 9.4, tvOS 16.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.9:macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 5.10:macOS 14.4, iOS 17.4, watchOS 10.4, tvOS 17.4, visionOS 1.1' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.0:macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.1:macOS 15.4, iOS 18.4, watchOS 11.4, tvOS 18.4, visionOS 2.4' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.2:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'SwiftStdlib 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'StdlibDeploymentTarget 6.3:macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, visionOS 9999' -Xfrontend -define-availability -Xfrontend 'SwiftCompatibilitySpan 5.0:macOS 10.14.4, iOS 12.2, watchOS 5.2, tvOS 12.2, visionOS 1.0' -Xfrontend -define-availability -Xfrontend 'SwiftCompatibilitySpan 6.2:macOS 26.0, iOS 26.0, watchOS 26.0, tvOS 26.0, visionOS 26.0' -Xfrontend -target-min-inlining-version -Xfrontend min -module-link-name swift_Concurrency -whole-module-optimization -color-diagnostics -parse-as-library -I S:/b/i686-unknown-windows-msvc/Runtime/./lib/swift/windows -o S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency//WINDOWS/i686/_Concurrency.obj @S:/b/i686-unknown-windows-msvc/Runtime/stdlib/public/Concurrency/f8648e79f94eeedf5418bb5702cbf65ec1a5cb5e.txt

1.      Swift version 6.2-dev (LLVM b0a5cdcad00b30f, Swift dd04814699722b8)

2.      Compiling with effective version 5.10

3.      While evaluating request IRGenRequest(IR Generation for module _Concurrency)

4.      While emitting IR SIL function "@$ss23withCheckedContinuation8function_xSS_yScCyxs5NeverOGXEtYalF".

 for '_unsafeInheritExecutor_withCheckedContinuation(function:_:)' (at S:/SourceCache/swift/stdlib/public/Concurrency/CheckedContinuation.swift:319:8)

Exception Code: 0x80000003

 #0 0x00007ff63b0198b5 HandleAbort S:\SourceCache\llvm-project\llvm\lib\Support\Windows\Signals.inc:372:0

 #1 0x00007ffb08bf1989 (C:\WINDOWS\System32\ucrtbase.dll+0xc1989)

 #2 0x00007ffb08bd4ab1 (C:\WINDOWS\System32\ucrtbase.dll+0xa4ab1)

 #3 0x00007ffb08bf2986 (C:\WINDOWS\System32\ucrtbase.dll+0xc2986)

 #4 0x00007ffb08bf2b61 (C:\WINDOWS\System32\ucrtbase.dll+0xc2b61)

 #5 0x00007ff63485343b `anonymous namespace'::emitIndirectAsyncFunctionPointer S:\SourceCache\swift\lib\IRGen\GenCall.cpp:2559:0

 #6 0x00007ff6348435f8 `swift::irgen::getAsyncFunctionAndSize'::`2'::<lambda_1>::operator() S:\SourceCache\swift\lib\IRGen\GenCall.cpp:2638:0

 #7 0x00007ff6348617df swift::irgen::getAsyncFunctionAndSize(class swift::irgen::IRGenFunction &, class swift::irgen::FunctionPointer, struct std::pair<bool, bool>) S:\SourceCache\swift\lib\IRGen\GenCall.cpp:2663:0

 #8 0x00007ff634848e21 llvm::Twine::{ctor} S:\SourceCache\llvm-project\llvm\include\llvm\ADT\Twine.h:279:0

 #9 0x00007ff634848e21 `anonymous namespace'::AsyncCallEmission::begin S:\SourceCache\swift\lib\IRGen\GenCall.cpp:3154:0

#10 0x00007ff63497aad6 `anonymous namespace'::IRGenSILFunction::visitFullApplySite S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:3857:0

#11 0x00007ff63496e7ea swift::SILInstructionVisitor<`anonymous namespace'::IRGenSILFunction,void>::visit S:\SourceCache\swift\include\swift\SIL\SILNodes.def:590:0

#12 0x00007ff634983e6c `anonymous namespace'::IRGenSILFunction::visitSILBasicBlock S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:2845:0

#13 0x00007ff63495faef `anonymous namespace'::IRGenSILFunction::emitSILFunction S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:2709:0

#14 0x00007ff63495eaee swift::irgen::IRGenModule::emitSILFunction(class swift::SILFunction *) S:\SourceCache\swift\lib\IRGen\IRGenSIL.cpp:2550:0

#15 0x00007ff6347a13de swift::irgen::CurrentIGMPtr::{dtor} S:\SourceCache\swift\lib\IRGen\IRGenModule.h:2061:0

#16 0x00007ff6347a13de swift::irgen::IRGenerator::emitGlobalTopLevel(class std::vector<class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>, class std::allocator<class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>>> const &) S:\SourceCache\swift\lib\IRGen\GenDecl.cpp:1235:0

#17 0x00007ff63472b5b1 std::vector<std::basic_string<char,std::char_traits<char>,std::allocator<char> >,std::allocator<std::basic_string<char,std::char_traits<char>,std::allocator<char> > > >::{dtor} C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.44.35207\include\vector:830:0

#18 0x00007ff63472b5b1 swift::IRGenRequest::evaluate(class swift::Evaluator &, struct swift::IRGenDescriptor) const S:\SourceCache\swift\lib\IRGen\IRGen.cpp:1406:0

#19 0x00007ff634734966 swift::SimpleRequest<class swift::IRGenRequest, (struct swift::IRGenDescriptor), 17>::evaluateRequest(class swift::IRGenRequest const &, class swift::Evaluator &) S:\SourceCache\swift\include\swift\AST\SimpleRequest.h:313:0

#20 0x00007ff6347253d7 swift::Evaluator::getResultUncached<class swift::IRGenRequest, class `class swift::GeneratedModule __cdecl swift::evaluateOrFatal<class swift::IRGenRequest>(class swift::Evaluator &, class swift::IRGenRequest)'::`2'::<lambda_1>>(class swift::IRGenRequest const &, class `class swift::GeneratedModule __cdecl swift::evaluateOrFatal<class swift::IRGenRequest>(class swift::Evaluator &, class swift::IRGenRequest)'::`2'::<lambda_1>) S:\SourceCache\swift\include\swift\AST\Evaluator.h:347:0

#21 0x00007ff63472eb91 swift::performIRGeneration(class swift::ModuleDecl *, class swift::IRGenOptions const &, struct swift::TBDGenOptions const &, class std::unique_ptr<class swift::SILModule, struct std::default_delete<class swift::SILModule>>, class llvm::StringRef, class swift::PrimarySpecificPaths const &, class llvm::ArrayRef<class std::basic_string<char, struct std::char_traits<char>, class std::allocator<char>>>, class llvm::GlobalVariable **) S:\SourceCache\swift\lib\IRGen\IRGen.cpp:1824:0

#22 0x00007ff63438de42 generateIR S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:1785:0

#23 0x00007ff63438de42 performCompileStepsPostSILGen S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:2139:0

#24 0x00007ff63438e3ca std::unique_ptr<swift::SILModule,std::default_delete<swift::SILModule> >::{dtor} C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.44.35207\include\memory:3426:0

#25 0x00007ff63438e3ca swift::performCompileStepsPostSema(class swift::CompilerInstance &, int &, class swift::FrontendObserver *) S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:777:0

#26 0x00007ff63438d42a performAction S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:1348:0

#27 0x00007ff63438d6db performCompile S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:1734:0

#28 0x00007ff63438f845 swift::performFrontend(class llvm::ArrayRef<char const *>, char const *, void *, class swift::FrontendObserver *) S:\SourceCache\swift\lib\FrontendTool\FrontendTool.cpp:2454:0

#29 0x00007ff6341cda5d run_driver S:\SourceCache\swift\lib\DriverTool\driver.cpp:443:0

#30 0x00007ff6341cd4b7 swift::mainEntry(int, char const **) S:\SourceCache\swift\lib\DriverTool\driver.cpp:530:0

#31 0x00007ff63b082b50 invoke_main D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:78:0

#32 0x00007ff63b082b50 __scrt_common_main_seh D:\a\_work\1\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288:0

#33 0x00007ffb0ac3e8d7 (C:\WINDOWS\System32\KERNEL32.DLL+0x2e8d7)

#34 0x00007ffb0b97c34c (C:\WINDOWS\SYSTEM32\ntdll.dll+0x3c34c)

<unknown>:0: error: compile command failed due to signal -2147483645 (use -v to see invocation)
```

### Expected behavior

Compiler does not crash.

### Environment

```
 swiftc -version
<unknown>:0: warning: using (deprecated) legacy driver, Swift installation does not contain swift-driver at: 'C:\Users\steel\src\toolchain\Program Files\Swift\Toolchains\0.0.0+Asserts\usr\bin\swift-driver-new.exe'
Swift version 6.2-dev (LLVM b0a5cdcad00b30f, Swift dd04814699722b8)
Target: x86_64-unknown-windows-msvc
```

### Additional information

_No response_

--- a/lib/IRGen/GenCall.cpp
+++ b/lib/IRGen/GenCall.cpp
@@ -2546,9 +2546,8 @@ llvm::Value *emitIndirectAsyncFunctionPointer(IRGenFunction &IGF,
   llvm::Constant *One =
       llvm::Constant::getIntegerValue(IntPtrTy, APInt(IntPtrTy->getBitWidth(),
                                                       1));
-  llvm::Constant *NegativeOne =
-      llvm::Constant::getIntegerValue(IntPtrTy, APInt(IntPtrTy->getBitWidth(),
-                                                      -2));
+  llvm::Constant *NegativeOne = llvm::Constant::getIntegerValue(
+      IntPtrTy, APInt(IntPtrTy->getBitWidth(), -2, /*isSigned*/ true));
   swift::irgen::Alignment PointerAlignment = IGF.IGM.getPointerAlignment();
 
   llvm::Value *PtrToInt = IGF.Builder.CreatePtrToInt(pointer, IntPtrTy);
@@ -2578,7 +2577,7 @@ llvm::Value *emitIndirectCoroFunctionPointer(IRGenFunction &IGF,
   llvm::Constant *One = llvm::Constant::getIntegerValue(
       IntPtrTy, APInt(IntPtrTy->getBitWidth(), 1));
   llvm::Constant *NegativeOne = llvm::Constant::getIntegerValue(
-      IntPtrTy, APInt(IntPtrTy->getBitWidth(), -2));
+      IntPtrTy, APInt(IntPtrTy->getBitWidth(), -2, /*isSigned*/ true));
   swift::irgen::Alignment PointerAlignment = IGF.IGM.getPointerAlignment();
 
   llvm::Value *PtrToInt = IGF.Builder.CreatePtrToInt(pointer, IntPtrTy);

